lib/src/request.dart

```diff
@@ -39,24 +39,14 @@ class Request {
             };
 
   /// [signature] is the [String] representation of the [Request]'s body.
-  String get signature =>
-      '$route/${method.value}/' +
-      sortedData(data) +
-      '/' +
-      sortedData(queryParameters) +
-      '/$headers';
+  String get signature => '$route/${method.value}/' + sortedData(data) + '/' + sortedData(queryParameters) + '/$headers';
```

Comment:
why make it on one line ?

Reply:
Yea this got formatted to 150 line length in the merge, my bad.